### PR TITLE
Update accordion content and styling to be semantic html(er)

### PR DESCRIPTION
**Pull request by gmrabian**
### Description

Remove all `<br>` tags and use paragraph in place of span when possible

(sorry the videos are not embedded but they were too small so please open them in a new tab)
|       | WP | SAR |
| --- | --- | ---- |
|  Admin  |  https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/12581b64-3cfe-44cd-9513-64ed909fd90e |  https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/fbcc2db0-c443-459f-8cb0-04f472f1818b |
|  State  |  https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/28996425-3546-4ff6-8c9a-17c6621ea525 |  https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/df150be0-61e5-48a9-ab96-ff6158979e7c |

How to fix space between span and admin instructions 🤔 
I wasn't sure how to get the link into the paragraph without span because the p tag separated each chunk onto a new line

### Related ticket(s)

CMDCT-3608

---
### How to test

Deployed url: https://d22pi53b5rbg35.cloudfront.net/
Check the browser inspector and verify there are no <br> tags

---
### Pre-review checklist

- ~[ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary~
- ~[ ] I have updated relevant documentation, if necessary~
- [x] I have performed a self-review of my code
- [x] I have manually tested this PR in the deployed cloud environment

### CODE REVIEW

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 1]
```diff
@@ -10,14 +10,26 @@ export default {
...
+            "This reporting tool is to be used to describe state or territory-specific initiatives. Start or amend a new MFP Work Plan below to capture these initiatives designed to increase the use and quality of home and community-based services.",
         },
         {
           type: "text",
-          as: "span",
-          content: "<br><strong>Admin Instructions</strong>",
+          as: "p",
+          content:
+            "<b> MFP Work Plan to Semi-Annual Progress Report Process</b>",
```
Comment (bangbay-bluetiger) [inline services/ui-src/src/verbiage/pages/accordion.ts:21]: I'd remove extra space between `<b>` and `MFP`.

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 2]
```diff
@@ -35,9 +47,21 @@ export default {
...
+            "This reporting tool is to be used to describe state or territory-specific initiatives. Start or amend a new MFP Work Plan below to capture these initiatives designed to increase the use and quality of home and community-based services.",
+        },
+        {
+          type: "text",
+          as: "p",
+          content:
+            "<strong> MFP Work Plan to Semi-Annual Progress Report Process </strong>",
```
Comment (bangbay-bluetiger) [inline services/ui-src/src/verbiage/pages/accordion.ts:58]: Trim spaces at ends.

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 3]
```diff
@@ -82,8 +106,8 @@ export default {
         },
         {
           type: "text",
-          as: "span",
-          content: "<br><br><b>Admin instructions</b>",
+          as: "p",
+          content: "<b>Admin instructions</b>",
```
Comment (bangbay-bluetiger) [inline services/ui-src/src/verbiage/pages/accordion.ts:110]: Seems like there's inconsistent usage of `<b>` vs `<strong>`. Most of these probably should be `<strong>` to indicate importance versus it being bolded just for style. But I'd leave it as-is for now..

Comment (gmrabian) [reply services/ui-src/src/verbiage/pages/accordion.ts:110]: Good point! Are they handled differently from an html or accessibility perspective? I guess I never considered why you might pick one over the other.

Comment (bangbay-bluetiger) [reply services/ui-src/src/verbiage/pages/accordion.ts:110]: In theory, screen readers can read those tags in different voices, but the tag differences are more to convey meaning. `<strong>` indicates "I want you to pay particular attention to this word/phrase" while `<b>` is more "I think this makes this word/phrase look better."

**Review by bangbay-bluetiger — APPROVED** [review]

File: services/ui-src/src/verbiage/pages/accordion.ts (hunk 3 above)
Comment (gmrabian) [reply services/ui-src/src/verbiage/pages/accordion.ts:110]: 💡good to know!

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 4]
```diff
@@ -10,14 +10,26 @@ export default {
         },
         {
           type: "text",
-          as: "span",
+          as: "p",
```
Comment (jessabean) [inline services/ui-src/src/verbiage/pages/accordion.ts:13]: A thing I learned while looking at this PR...I am not sure we need this `as: "p"` line! The parser / `<Text>` component defaults to a p tag.

**Review by jessabean — APPROVED** [review]
🙌🏻

**Pull request merged**